HRIS.Services.Tests/Services/EmployeeAddressServiceUnitTest.cs

```diff
@@ -4,6 +4,7 @@
 using Moq;
 using RR.Tests.Data;
 using RR.UnitOfWork;
+using RR.UnitOfWork.Entities.ATS;
```

Comment:
Refactor: we don't need these lines:
using HRIS.Services.Interfaces;
using RR.UnitOfWork.Entities.ATS;

Reply:
Okay cool
Will add the auth stuff